src/config/loadGitHubAppsFromEnvironment.ts

```diff
@@ -1,13 +1,68 @@
 import { AppAuthStrategyOptions } from '@/types';
 
+export class GitHubApp {
+  org: string;
+  auth: AppAuthStrategyOptions;
+  project;
+
+  constructor(org: string, opts) {
+    this.org = org;
+    this.auth = {
+      appId: Number(opts.GH_APP_IDENTIFIER),
+      privateKey: opts.GH_APP_SECRET_KEY.replace(/\\n/g, '\n'),
+    };
+    this.project = {
```

Comment:
guessing for codecov an if conditional will be introduced here?

Reply:
I was thinking of moving the defaults out of our code and into envvars.